.github/workflows/build.yml

```diff
@@ -22,10 +22,10 @@ jobs:
       run: cargo build --verbose
       continue-on-error: ${{ matrix.toolchain != 'stable' }}
     - name: Run tests
-      run: cargo test --verbose
+      run: cargo test --all --verbose
       continue-on-error: ${{ matrix.toolchain != 'stable' }}
     - name: Run tests (vendored-openssl)
-      run: cargo test --features vendored-openssl --verbose
```

Comment:
clippy should also get an `--all`.  rustfmt does not need it though (its weird and they don't want to conform)

Reply:
Good catch, done